repository-meta-analyzer/src/main/java/org/acme/repositories/NugetMetaAnalyzer.java

```diff
@@ -49,6 +51,9 @@ public class NugetMetaAnalyzer extends AbstractMetaAnalyzer {
             new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX"),
             new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'")
     };
+
+    @Inject
+    MetaModel meta;
```

Comment:
`MetaModel` should not be injected. Instead, create new instances in `analyze`, like it was done before.